lib/core/src/lib/form/components/widgets/date-time/date-time.widget.spec.ts

```diff
@@ -52,7 +52,7 @@ describe('DateTimeWidgetComponent', () => {
     });
 
     it('should setup min value for date picker', () => {
-        const minValue = '1982-03-13T10:00:000Z';
+        let minValue = '1982-03-13T10:00:000Z';
```

Comment:
should be const as you are not changing it

Reply:
changed back to const